songbird/playlist.py

```diff
@@ -1,31 +1,37 @@
 from __future__ import annotations
 
+import json
 from asyncio import create_subprocess_shell
 from asyncio.subprocess import PIPE
-import json
-from typing import Any, Dict, List
+from typing import (
+    Any,
+    Generator,
+)
 
 from songbird.helpers import ytdl
 from songbird.songbird import Metadata
 
 
-async def get_playlist(playlist: str) -> List[YoutubeVideo]:
+async def get_playlist(playlist: str) -> list[YoutubeVideo]:
     cmd = f"yt-dlp '{playlist}' --flat-playlist --dump-single-json"
 
     proc = await create_subprocess_shell(cmd, stdout=PIPE, stderr=PIPE)
-    stdout, _ = await proc.communicate()
+    stdout, stderr = await proc.communicate()
 
-    out: List[YoutubeVideo] = []
+    if stderr:
+        raise ValueError(stderr)
+
+    out: list[YoutubeVideo] = []
     if stdout:
         out.extend(
             YoutubeVideo(entry) for entry in json.loads(stdout)["entries"]
-        )
+        )  # TODO: Errors with youtube-dl outs
 
     return out
 
 
 class YoutubeVideo:
-    def __init__(self, video: Dict[str, Any]) -> None:
+    def __init__(self, video: dict[str, Any]) -> None:
```

Comment:
captial `Dict` and `List` etc should still be used

Reply:
It looks much prettier, python 3.11 style

Reply:
https://www.python.org/dev/peps/pep-0563/